app/views/admin/_uservoice_widget.html.erb

```diff
@@ -25,24 +25,19 @@ UserVoice.push(['set', {
 // Identify the user and pass traits
 // To enable, replace sample data with actual user traits and uncomment the line
 UserVoice.push(['identify', {
```

Comment:
Actually, nearly all parameters here are new, so pass them as hash from controller. Views shouldn't know anything of services.